src/colp/components/geospatial_check.py

```diff
@@ -0,0 +1,53 @@
+import streamlit as st
+import pandas as pd
+import json
+
+
+class GeospatialCheck:
+    def __init__(self, data):
+        self.geospatial_check = data["geospatial_check"]
+
+    def __call__(self):
+        st.subheader("Geospatial Check")
+        if self.geospatial_check is None:
+            st.info("Geospatial Check Table Not Found")
+            return
+        df = self.geospatial_check
+        df["result"] = df["result"].apply(json.loads)
+        all_records = df.to_dict("records")
+        self.display_not_in_nyc(all_records[0])
+        self.display_nonmatch(all_records[1])
+
+    def cast_version(self, v):
+        if len(v) == 6:
+            return str(v[:4]) + "-" + str(v[4:])
+
+    def display_not_in_nyc(self, records):
+        st.markdown("#### Properties That Are Not in NYC Borough Boundaries")
+        records = records["result"][0]["values"]
```

Comment:
I think we could add text between the header and the plot the way the **Properties That Have Inconsistent Geographies** report does

Reply:
> I think this is good to be merged in as is. I have a couple comments but they aren't necessary. Did you decide whether to add the expected borough from bbl/cd columns?

Yea, we can do that. It is a great suggestion. We need to implement that on the COLP side. I can take a look.

Reply:
ok sick! might be good to link the issue in this convo